test/data/v1beta1/buildrun_buildah_cr_local_source_upload.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: shipwright.io/v1beta1
+kind: BuildRun
+metadata:
+  name: buildah-golang-local-source-upload
+spec:
+  build:
+    name: buildah-local-source-upload
+  sources:
+    - name: local-copy
+      type: LocalCopy
```

Comment:
Must be `.spec.source`. And I just noticed that this is missing in Beta. But there we must support the LocalCopy type.